src/manifest.json

```diff
@@ -28,5 +28,17 @@
     "https://cmu-student-government.github.io/",
     "activeTab",
     "storage"
-  ]
+  ],
+
+  "applications": {
+    "gecko": {
+      "id": "cmunits-gecko@sterdam.com",
```

Comment:
If this is supposed to be any sort of a contact email, please use StuGov CTO email: stugov-cto@andrew.cmu.edu

Reply:
I don't think this is supposed to be an actual email address, as per Mozilla's documentation it should be "A string formatted like an email address: extensionname@example.org" and "Be aware that using a real email address here may attract spam".
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings